Tasks/VsTestV2/task.loc.json

```diff
@@ -18,7 +18,7 @@
   "version": {
     "Major": 2,
     "Minor": 210,
```

Comment:
Could you please bump the task to the current sprint? (should be 214.0)

Reply:
Done